scss/_patterns_navigation.scss

```diff
@@ -287,24 +286,28 @@ $lightness-threshold: 70;
   // color for navigation background
   $color-navigation-background,
   // color for navigation text
-  $color-navigation-text
+  $color-navigation-text,
+  // transparent color of selected and hovered menu item
+  $color-background-hover
```

Comment:
Is this used anywhere?

Reply:
I've picked the wrong choice during conflict resolution, removed now.